src/README.md

```diff
@@ -1,101 +1,127 @@
+![](/zilla-overview.png)
 
-You can see a working [Petstore Demo](https://github.com/aklivity/zilla-demos/tree/main/petstore) using OpenAPI/AsyncAPI schemas. Zilla can also define and proxy MQTT endpoints using a pair of AsyncAPI schemas. Check out the [Taxi Demo](https://github.com/aklivity/zilla-demos/tree/main/taxi) to see a Zilla MQTT proxy defined using AsyncAPI, which is deployed [Live](https://taxi.aklivity.io/) using Kubernetes.
+::: info Just want to build?
+Jump to the [quick start](./how-tos/quickstart.md) guide.
+:::
 
-You can explicitly define your APIs in a Zilla configuration by carefully orchestrating all of the different [Bindings](./concepts/config-intro.md#Bindings) Zilla has to offer. You can see many of them on display by using the [Kafka Proxy Quickstart](./tutorials/quickstart/kafka-proxies.md) or checking out the [Zilla Examples](https://github.com/aklivity/zilla-examples) repo.
+## Why Zilla?
 
-## Running Zilla via Docker
+Some of the hardest operational challenges inside distributed architectures relate to networking and observability. As a result, edge and service proxies are required to achieve resilient, transparent, and properly routed connectivity among distributed services.
 
-Run the latest Zilla release with the default empty configuration via docker.
+While solutions such as Envoy, HAProxy, NGINX, etc. can help unify a distributed data plane, they are designed for “mesh” deployments in which services are directly interconnected and communicate almost exclusively over HTTP. Inside event-driven architectures (EDAs) though, services are separated by an event/message broker, and multiple protocols are present. This decoupled and multi-protocol nature of EDAs presents a new class of networking, observability, and security challenges that the Zilla proxy is designed to address.
 
-```bash:no-line-numbers
-docker run ghcr.io/aklivity/zilla:latest start -v
-```
+## Zilla Features
 
-The output should display the Zilla config and `started` to know Zilla is ready for traffic.
+### Self-Contained, Stateless Architecture
 
-```output:no-line-numbers
-// default Zilla config
-name: default
+Zilla has no external dependencies, is stateless, and is highly memory efficient. When deployed as an edge proxy, it scales horizontally to support millions of concurrently connected clients.
 
-// Zilla status
-started
-```
+### HTTP/2, gRPC, SSE, MQTT, and Kafka Support + Protocol Mediation
 
-Specify your own `zilla.yaml` file.
+Inside Zilla, every protocol, whether it is TCP, TLS, HTTP, Kafka, gRPC, etc., is treated as a stream, so mediating between protocols simplifies to mapping protocol-specific metadata.
 
-```bash:no-line-numbers
-docker run -v ./zilla.yaml:/etc/zilla/zilla.yaml ghcr.io/aklivity/zilla:latest start -v
-```
+### Protobuf, Avro, OpenAPI, and AsyncAPI Support
```

Comment:
Do these belong together?

I realize there is implementation overlap in that Apicurio can host all the different types of artifacts for JSON schema, Protobuf schema, Avro schema, as well as OpenAPI specs and AsyncAPI specs, but IMHO they feel sufficiently different to highlight separately.

Also, we should highlight JSON schemas here too.

Reply:
Optional change:

```

### Protobuf, Avro, and JSON Schema payloads

Zilla supports payload schema specifications for message validation and translation.

### OpenAPI and AsyncAPI Support

Zilla supports different API schema specifications for message validation and API creation.
```

Reply:
I'm fine with splitting these as per AJ's proposal.

Reply:
@jfallows pushed, let me know if that looks good.

Reply:
LGTM2. 👍